pkg/client/forwarder.go

```diff
@@ -84,10 +85,16 @@ func checkLine(l string, prefixes []string) checkResult {
 		return checkResult{skip: true}
 	}
 	msg := parts[1]
-	for _, prefix := range prefixes {
+	for _, prefix := range dropPrefixes {
 		if strings.HasPrefix(msg, prefix) {
 			m := strings.TrimPrefix(msg, prefix)
-			return checkResult{skip: false, messageWithoutPrefix: m, ts: ts}
+			return checkResult{skip: false, messageWithoutPrefix: m + " ACTION=Drop", ts: ts}
```

Comment:
I would prefer to have all lowercase values, e.g. `ACTION=drop`, key should remain all uppercase to match all other fields.